test/cpu_debugger.cpp

```diff
@@ -38,8 +38,14 @@
 using namespace ngraph;
 using namespace std;
 
+// These tests are for DEX mode only.
 TEST(debugger, add_breakpoint)
 {
+    if (std::getenv("NGRAPH_CODEGEN") != nullptr &&
+        std::string(std::getenv("NGRAPH_CODEGEN")) != "0")
+    {
+        return;
```

Comment:
My only concern is that now we are returning PASSED when the test wasn't actually run. Is there a way to just ignore the test based on NGRAPH_CODEGEN? I did a quick search and found GTEST_SKIP. Maybe it's worth having a look.

Reply:
Thank you. GTEST_SKIP was introduced last year. But our gtest is older version and does not have it. Need to update gtest.

Reply:
The latest gtest release dates back to Aug 31, 2018, which doe not have GTEST_SKIP. Will create another PR when there is a new release.

Reply:
What could we do in the meantime? Should we raise an assert saying that the test is not supported in codegen mode better than return pass? I feel that if we return pass when then test didn't actually run we will lose track of which tests actually pass in codegen mode.